--- menu_data/aqemu.desktop.orig	2009-06-04 17:29:38.000000000 +1000
+++ menu_data/aqemu.desktop	2011-08-30 22:59:02.621556022 +1000
@@ -1,14 +1,12 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=AQEMU
 Exec=aqemu
 Terminal=false
 Type=Application
-Icon=aqemu_48x48.png
+Icon=aqemu
 StartupNotify=true
 Categories=System;Emulator;
 
+Name=AQEMU
+Comment=Virtual machine manager
 Name[ru]=AQEMU
-Name[en]=AQEMU
 Comment[ru]=Управление виртуальными машинами
-Comment[en]=Virtual machine manager
